internal/website/gatherexamples/gatherexamples.go

```diff
@@ -14,22 +14,24 @@
 
 // Command gatherexamples extracts examples in a Go module into a JSON-formatted
 // object. This is used as input for building the Go CDK Hugo website.
-// Examples must include a comment that starts with "// This example is used in"
+//
+// Examples must include a comment
+// "// PRAGMA(gocloud.dev): Package this example for gocloud.dev."
```

Comment:
Your PR description mentions this for "Variables set up elsewhere:", not "This example is used in"

Reply:
Oops, fixed.